<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>szukajUbezpieczenia</class>
 <widget class="QDialog" name="szukajUbezpieczenia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="szukajButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>470</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Szukaj</string>
   </property>
  </widget>
  <widget class="QPushButton" name="wrocButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>470</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Wróć</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>361</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QCheckBox" name="ubezpieczycielCB">
      <property name="text">
       <string>Ubezpieczyciel</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QLineEdit" name="numerRejLE"/>
    </item>
    <item row="3" column="0" colspan="2">
     <widget class="QLineEdit" name="numerUbkLE"/>
    </item>
    <item row="7" column="1">
     <widget class="QDoubleSpinBox" name="doKosztDoubleSpinBox">
      <property name="prefix">
       <string notr="true"/>
      </property>
      <property name="suffix">
       <string notr="true"/>
      </property>
      <property name="maximum">
       <double>999999.989999999990687</double>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QDoubleSpinBox" name="dokwotaDoubleSpinBox">
      <property name="maximum">
       <double>1000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QDateEdit" name="doDataZakDateEdit">
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QCheckBox" name="odDataZakCB">
      <property name="text">
       <string>Od daty zakończenia</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QCheckBox" name="doDataZawCB">
      <property name="text">
       <string>Do daty zawarcia umowy</string>
      </property>
     </widget>
    </item>
    <item row="10" column="0">
     <widget class="QCheckBox" name="odDataZawCB">
      <property name="text">
       <string>Od daty zawarcia umowy</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QCheckBox" name="odKwotaUbCB">
      <property name="text">
       <string>Od kwoty ubezpieczenia</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QCheckBox" name="doDataZakCB">
      <property name="text">
       <string>Do daty zakńczenia</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QCheckBox" name="numerUbCB">
      <property name="text">
       <string>Numer ubezpieczenia</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QCheckBox" name="doKosztCB">
      <property name="text">
       <string>Koszt do</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0">
     <widget class="QDateEdit" name="odDataZawDateEdit">
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QCheckBox" name="doKwotaUbCB">
      <property name="text">
       <string>Do kwoty ubezpieczenia</string>
      </property>
     </widget>
    </item>
    <item row="13" column="0">
     <widget class="QDateEdit" name="odDataZakDateEdit">
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QDoubleSpinBox" name="odkwotaDoubleSpinBox">
      <property name="maximum">
       <double>1000000.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QDoubleSpinBox" name="odKosztDoubleSpinBox">
      <property name="prefix">
       <string notr="true"/>
      </property>
      <property name="suffix">
       <string notr="true"/>
      </property>
      <property name="maximum">
       <double>999999.989999999990687</double>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QCheckBox" name="odKosztCB">
      <property name="text">
       <string>Koszt od</string>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QDateEdit" name="doDataZawDateEdit">
      <property name="displayFormat">
       <string>yyyy-MM-dd</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QCheckBox" name="numerRejCB">
      <property name="text">
       <string>Numer rejestracyjny</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QLineEdit" name="ubezpieczycielLE"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
